scripts/gen-edpm-baremetal-kustomize.sh

```diff
@@ -129,6 +129,9 @@ patches:
     - op: replace
       path: /spec/nodeTemplate/ansibleSSHPrivateKeySecret
       value: ${EDPM_ANSIBLE_SECRET}
+    - op: replace
+      path: /spec/nodeTemplate/ansible/ansibleUser
+      value: ${EDPM_ANSIBLE_USER:-"cloud-admin"}
```

Comment:
Changing this won't work. `ansibleUser` and `cloudUserName` should be changed together. https://github.com/openstack-k8s-operators/install_yamls/pull/537